controllers/authentication/union_handler_test.go

```diff
@@ -1,4 +1,4 @@
-// SPDX-FileCopyrightText: SAP SE or an SAP affiliate company and Gardener contributors
+// SPDX-FileCopyrightText: 2021 SAP SE or an SAP affiliate company and Gardener contributors
```

Comment:
Should we remove the year here as well?
```suggestion
// SPDX-FileCopyrightText: SAP SE or an SAP affiliate company and Gardener contributors
```